evm_loader/program/src/error.rs

```diff
@@ -44,18 +44,27 @@ pub enum Error {
     #[error("Account {0} - blocked")]
     AccountBlocked(Address),
 
-    #[error("Account {0} - invalid tag {1}, expected {2}")]
-    AccountInvalidTag(Pubkey, u8, u8),
+    #[error("Account {0} - invalid tag, expected {1}")]
```

Comment:
Why do you remove actual tag value from the error message? It is helpful to understand which account was passed on that place.

Reply:
It doesn't really needed, and can be extracted from the account when it does.
It makes code less readable because of autoformat tools and inconsistency with other similar error messages